pyccl/halos/profiles.py

```diff
@@ -989,31 +1191,39 @@ def _integ_interp(self):
         from scipy.integrate import quad
 
         def integrand(x):
-            return self._form_factor(x)*x
+            return self._form_factor(x) * x
 
         q_arr = np.geomspace(self.qrange[0], self.qrange[1], self.nq)
         # We use the `weight` feature of quad to quickly estimate
         # the Fourier transform. We could use the existing FFTLog
         # framework, but this is a lot less of a kerfuffle.
-        f_arr = np.array([quad(integrand,
-                               a=1e-4, b=self.x_out,  # limits of integration
-                               weight="sin",  # fourier sine weight
-                               wvar=q)[0] / q
-                          for q in q_arr])
-        Fq = interp1d(np.log(q_arr), f_arr,
-                      fill_value="extrapolate",
-                      bounds_error=False)
+        f_arr = np.array(
+            [
+                quad(
+                    integrand,
+                    a=1e-4,
+                    b=self.x_out,  # limits of integration
+                    weight="sin",  # fourier sine weight
+                    wvar=q,
+                )[0]
+                / q
+                for q in q_arr
+            ]
+        )
+        Fq = interp1d(
+            np.log(q_arr), f_arr, fill_value="extrapolate", bounds_error=False
+        )
```

Comment:
And this is also potentially hard to read. But let's see what the other reviewers say as well. The point is that, in Python, we don't care as much about the bracket alignment as in C-like languages, so it's fine if we just put them at the end. The function names are self-explanatory, so it would still be easy for the reader to understand what's going on.